core/services/job/job_orm_test.go

```diff
@@ -1880,16 +1877,15 @@ func Test_ORM_FindJobByWorkflow(t *testing.T) {
 				ds: pgtest.NewSqlxDB(t),
 			},
 			args: args{
-				spec: job.WorkflowSpec{
-					ID:            1,
-					WorkflowID:    "workflow 4",
-					Workflow:      "anything",
-					WorkflowOwner: "me",
-					WorkflowName:  "wf4",
+				spec: &job.WorkflowSpec{
+					ID:       1,
+					Workflow: pkgworkflows.WFYamlSpec(t, "workflow03", "0x0123456789012345678901234567890123456789"),
 				},
-				before: func(t *testing.T, o job.ORM, s job.WorkflowSpec) int32 {
-					s.WorkflowOwner = "not me"
-					return mustInsertWFJob(t, o, s)
+				before: func(t *testing.T, o job.ORM, s *job.WorkflowSpec) int32 {
+					var c job.WorkflowSpec
+					c.ID = s.ID
+					c.Workflow = pkgworkflows.WFYamlSpec(t, "workflow03", "0xabcdefabcdefabcdefabcdefabcdefabcdefabcd") // insert with owner "0xabcdefabcdefabcdefabcdefabcdefabcdefabcdefabcd" instead of "0x0123456789012345678901234567890123456789"
```

Comment:
I typically use constants for this, which would allow you to skip these comments 🙂